cmd/mindthegap/create/bundle/bundle.go

```diff
@@ -311,7 +352,7 @@ func pullImages(
 						return err
 					}
 
-					if err := remote.WriteIndex(ref, imageIndex, destRemoteOpts...); err != nil {
+					if err := remote.Push(ref, image, destRemoteOpts...); err != nil {
```

Comment:
I'm unsure whether this change will write referenced images in indexes:

godoc for `WriteIndex`:

> WriteIndex pushes the provided ImageIndex to the specified image reference.
> WriteIndex will attempt to push all of the referenced manifests before
> attempting to push the ImageIndex, to retain referential integrity.

godoc for `Push`:

> Push uploads the given Taggable to the specified reference.

Are we sure that `Push` will handle multiplatform image indexes correctly?

Reply:
The `WriteIndex` calls the `Push` with no parameters modifications - https://github.com/google/go-containerregistry/blob/v0.20.3/pkg/v1/remote/write.go#L648-L653

Reply:
OK it seems to be working - I'm not sure how but I can look at that separately.

Reply:
@mhrabovcin No need to change this - thanks for looking into the code!